### PR TITLE
refactor: use webp images for smaller size

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

### CODE REVIEW

File: public/img/products/eth.png
Comment (kelsos) [inline public/img/products/eth.png:1]: Shouldn't this be the same size as the original?

Comment (lukicenturi) [reply public/img/products/eth.png:1]: So the `eth.png` one isn't used at all.
The `eth@2x.png` is being used, so I remove the old `eth.png` and rename `eth@2x.png` to `eth.png`

Comment (kelsos) [reply public/img/products/eth.png:1]: I see, thanks for the clarification :)

**Review by kelsos — APPROVED** [review]
lgtm thanks @lukicenturi

**Pull request merged**